checkmarx/util.py

```diff
@@ -397,9 +425,18 @@ def download_repo_and_create_scan(
         attribute_key=model.CustomFieldKeys.COMPONENT_NAME,
         value=artifact_name,
     )
-
+    cx_project.project_details.set_custom_field(
+        attribute_key=model.CustomFieldKeys.VERSION,
+        value=artifact_version,
+    )
     cx_project.update_remote_project()
-    scan_id = cx_project.start_scan()
+
+    scan_settings = model.ScanSettings(
+        projectId=cx_project.project_details.id,
+        comment=f'Scanning artifact name: {artifact_name}, version: {artifact_version}, '
+            f'commit: {hash}'
```

Comment:
does this work?
I'd expect 
```suggestion
        comment=(
            f'Scanning artifact name: {artifact_name}, version: {artifact_version}, '
            f'commit: {hash}'
        ),
```
to be required here.

Reply:
>>> comment = 'foo' ' bar'
>>> print(comment)
foo bar
see: https://docs.python.org/3/reference/lexical_analysis.html#string-literal-concatenation